app/controllers/brexit_landing_page_controller.rb

```diff
@@ -2,7 +2,10 @@ class BrexitLandingPageController < ApplicationController
   include Slimmer::Headers
 
   skip_before_action :set_expiry
-  before_action -> { set_expiry(1.minute) }
+  before_action do
+    content_item = ContentItem.find!(request.path)
```

Comment:
In the best case this is making a call to the cache (memcached ?) so it's not super expensive, but if we're calling the cache twice for every request I think we can do better. In the worst case and the cache is unavailable this doubles the calls to content store.

Reply:
Think I've fixed this